testRearrangeCarGroup8.py

```diff
@@ -0,0 +1,118 @@
+import unittest
+import numpy as np
+from RearrangeCarGroup8 import RearrangeCars
+
+class TestRearrangeCars(unittest.TestCase):
+
+    '''Unittests for RearrangeCars class and promised fuctionality
+    '''
+
+    def setUp(self):
+
+        '''Basic setting that will be used in several tests
+        '''
+        
+        self.initial_car_order = [1, 2, 0, 3]
+        self.final_car_order = [3, 1, 2, 0]
+        self.rc = RearrangeCars(list(self.initial_car_order), list(self.final_car_order))
+
+    def test_empty_parking_lot(self):
+
+        '''Test the case when the parking lot is empty,

```

Comment:
Hi Carmen, here is the information I found.
The parameter is a verbosity level. There are 3 different levels:
- 0 (quiet): you just get the total numbers of tests executed and the global result
- 1 (default): you get the same plus a dot for every successful test or a F for every failure
- 2 (verbose): you get the help string of every test and the result